api/apps/api/src/modules/access-control/projects-acl/project-acl.service.ts

```diff
@@ -92,4 +93,53 @@ export class ProjectAclService implements ProjectAccessControl {
       this.canViewSolutionRoles,
     );
   }
+
+  async checkUserIsOwner(userId: string, projectId: string): Promise<void> {
+    const userIsProjectOwner = await this.roles.findOne({
+      where: {
+        projectId,
+        userId,
+        roleName: Roles.project_owner,
+      },
+    });
+    if (!userIsProjectOwner) {
+      throw new ForbiddenException();
+    }
+  }
+
+  async findUsersInProject(
+    projectId: string,
+  ): Promise<UsersProjectsApiEntity[]> {
+    const usersInProject = await this.roles.find({
+      where: { projectId },
+      select: ['roleName'],
+      relations: ['user'],
+    });
+    return usersInProject;
+  }
```

Comment:
As we are peeking on `user` relation, we are close to https://refactoring.guru/pl/refactoring/smells/couplers smell.
Let's ask ourselves a question: Why would AccessControl need to know user details? It pollutes its boundaries, doesn't it?

Reply:
@kgajowy on one hand, I understand why you say that - it would be ideal to keep concepts (modules) as separated as possible. Additionally, there's a possibility in the future that we'll introduce `access-control` to the `users` module, and that might cause circular dependencies in the future.

On the other hand, if we don't return _any_ user information here, then the FE will not be able to build a proper list of project contributors (at least not without making unnecessary requests to find the user information). Also, the purpose of `access-control` is to grant roles on things to _users_ - so one could argue that it needs to know user details...

One idea that might be a good common ground to address both concerns would be do a plain old `left join` with the `users` table, and just pick the user `name` (and other fields the FE might need) in a query builder. This way, we would prevent from having an explicit dependency between the `access-control` module, but we would still return all the data the FE needs to properly render this list - what are your thoughts about this?

Reply:
We can either: 
* return just the ids and compose the results on `project` module
* make a compromise - i.e. not mess with modules, but make a dedicated read model, utilizing both tables (could be a db view) - such module would use both (indirectly) just for reads

Reply:
The main aspect is there to not extend module just because the view needs something - we want it to be "clean" (purist alert) and do what we designed it for.

Reply:
Correct me if I'm wrong (that I will probably be) but are we actually extending the module here instead of just accessing the joined table for `users`, `projects`  and `roles` and just adding some information from there?

I mean, the thing I did (now) was use a `createQueryBuilder` method to add only the information needed by FE to do some kind of listing of users (displayName, id, avatarDataUrl), and maybe even the id can be skipped.

Reply:
I believe it would be a standalone module

Reply:
@rubvalave please add a `/** @debt */` comment to the docs of this method, explaining what the problem is, how we plan to tackle it, and let's refactor this later down the road once we revisit this module after the demo. 

This will resurface once we tackle organization/scenario/solution permissions, so let's merge this PR for now and tackle this issue after the demo :+1: